Source/Integrations/Kysect.Shreks.GoogleIntegration/Tools/GoogleSheetCreator.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Google.Apis.Sheets.v4;
+using Google.Apis.Sheets.v4.Data;
+using Kysect.Shreks.GoogleIntegration.Extensions;
+using Kysect.Shreks.GoogleIntegration.Models;
+using Kysect.Shreks.GoogleIntegration.Sheets;
+
+namespace Kysect.Shreks.GoogleIntegration.Tools;
+
+public class GoogleSheetCreator
+{
+    private readonly SheetsService _service;
+    private readonly string _spreadsheetId;
+
+    public GoogleSheetCreator(SheetsService service, string spreadsheetId)
+    {
+        _service = service;
+        _spreadsheetId = spreadsheetId;
+    }
+
+    public async Task<T> GetOrCreateSheetAsync<T>(CancellationToken token)
```

Comment:
```suggestion
    public async Task<T> GetOrCreateSheetAsync<T>(CancellationToken token = default)
```

ты же до этого писал дефолт - где-то он есть, а где-то нет..

Reply:
Ну default есть только в методах, которые нужны для работы с GoogleTableAccessor, из них он прокидывается во все остальные